app/templates/navbar.html

```diff
@@ -0,0 +1,22 @@
+<nav class="navbar navbar-default" role="navigation">
+    <div class="container">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle" ng-init="navCollapsed = true" ng-click="navCollapsed = !navCollapsed">
```

Comment:
Even small controller is better that init, what i in the future someone will be want to add something to this...